tests/main/snapd-homedirs-vendored/task.yaml

```diff
@@ -18,11 +18,11 @@ prepare: |
     useradd -b /remote/users -m -U "$USERNAME"
 
     # Download current snapd edge
-    snap download --edge snapd --basename=snapd_edge
+    snap download --stable snapd --basename=snapd_stable
```

Comment:
Intended to fix 
![image](https://github.com/user-attachments/assets/ffabf103-a2eb-43f8-b6ba-099066ed7af3)